beakerx/gradlew

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
beakerx subdirectory should not be handled by gradle, see the proposed cmds for readme?
i will IM to explain.